constraints/build.gradle

```diff
@@ -12,6 +13,13 @@ test {
   useJUnitPlatform()
 }
 
+jacocoTestReport {
+  dependsOn test
+  reports {
+    xml.enabled true
```

Comment:
Thanks for getting started on this. I think we need to update:

```
reports {
  xml.enabled true
}
```

Since it is throwing:

```text
Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.6/userguide/command_line_interface.html#sec:command_line_warnings
```

Which upon further inspection gives:

```text
> Configure project :contrib
The Report.enabled property has been deprecated. This is scheduled to be removed in Gradle 8.0. Please use the required property instead. See https://docs.gradle.org/7.6/dsl/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:enabled for more details.
        at build_a2tk82l1csrm9c80cb761cmeg$_run_closure3$_closure7.doCall(/Users/ccamargo/Projects/aerie/contrib/build.gradle:23)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
```

Reply:
I've replaced `xml.enabled true` with `xml.required = true` :+1:

We don't need the xml report for our current release, but we may want it if we want to generate our own html reports.